src/main/java/org/cyclonedx/gradle/CycloneDxTask.java

```diff
@@ -336,6 +345,23 @@ public void setLicenseChoice(final Consumer<LicenseChoice> customizer) {
         getInputs().property("LicenseChoice", licenseChoice);
     }
 
+    public void setVCSGit(Consumer<ExternalReference> customizer) {
```

Comment:
Codebase in this project uses `final` keywords whenever possible. There are few places here where `final` is missing. Unfortunately we don't have automated check here, because with Java 8 it's little bit tricky nowadays.